Add an intro to plugins and the plugin manager (#37)
napari/napari-workshop-template: napari-workshops/plugins.md

* Provide sample data
* Change the look of napari with a theme

[The napari hub](https://www.napari-hub.org) is a repository for searching/discovering plugins.
Plugins can be installed as Python packages using conda or pip as normal or from within the
napari GUI using the Plugin menu.

```{note}
The built-in [napari Plugin Manager](https://napari.org/napari-plugin-manager/) by default will attempt to 
install plugins in the same fashion that napari was installed. This means that if you installed napari 
via `conda`, it will attempt to install plugins via `conda` and if your napari was installed via `pip`, 
it will attempt to install plugins via `pip`. You can control this using the `Installation Info` dropdown 
in the Plugin Manager.  

Also keep in mind that plugins with complex or compiled dependencies or those that require non-Python components may not install properly. In those cases, you will need to check the documentation of the plugin for information regarding installation.
```

For more information on plugins, please see the [official Plugin documentation](https://napari.org/stable/plugins/index.html).

## Install Your First Plugins  

> * Navigate to [napari-hub.org](https://www.napari-hub.org)—this is the platform to discover and share plugins.  

Reviewer: ```suggestion
* (Optional) Navigate to [napari-hub.org](https://www.napari-hub.org)—this is the platform to discover and share plugins. Actual plugin installation will be done via napari--we detail this step below.
```

    * Select: **File**>**Open Sample**>**napari-bio-sample-data**>**3D nuclei**  

    This sample data includes the same `nuclei` data as before, but this time a 3D labels and surface layer
    are present. You can explore these in the 3D view. Delete the surface layer once you're ready to move
    on.
* Open **Tools**>**Measurement Tables**>**Regionprops (scikit-image, nsr)**
    * Ensure that `nuclei (data)` is selected for the `image` layer and `nuclei label (data)` is selected
    for the `labels` layer.
    * Use the checkboxes to select some properties to measure. Refer to the [scikit-image `measure.regionprops` documentation](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops) for more information
    * Click run, which should yield a table of measurements—depending how many you checked, this could take
    a moment to run.

    ```{tip}
    Grab the titlebar of the new widget `Properties of nuclei label` and drag it off the napari
    viewer to undock. Then you can resize the table for easier browsing.
    You can re-dock the widget by dragging it on the existing side dock  or the bottom of the viewer,

>     e.g. where the viewer buttons are, to dock it beneath the canvas.

Author: Yeah this isn't working, it's tricky. I made a new commit to better describe it. works for me to drag the widget title bar from below the viewer onto the canvas.

>     e.g. where the viewer buttons are, to dock it beneath the canvas.

Reviewer: ```suggestion
    e.g. where the viewer buttons are, to dock it as a tab next to "layer list".
```

>     e.g. where the viewer buttons are, to dock it as a tab next to "layer list".

Author: ```suggestion
    by dragging the widget title bar from below the viewer onto the canvas (left of the `Activity` button).
```
